home-work-angular/app-cat/directives/ca.foucs.directive.js

```diff
@@ -0,0 +1,22 @@
+(function (module) {
+    'use strict';
+    function itemFocus() {
+        return {
+            restrict: 'A',
+            scope: {
+                itemFocus: '=itemFocus'
+            },
+            link: function (scope, elem, attr, ctrl) {
+                scope.$watch('itemFocus', focusElement);
```

Comment:
is the watcher on directive or the scope property, better to rename scope property.